lib/matplotlib/axes/_axes.py

```diff
@@ -3190,6 +3190,19 @@ def grouped_bar(self, heights, *, positions=None, group_spacing=1.5, bar_spacing
 
             If not specified, the colors from the Axes property cycle will be used.
 
+        hatch : sequence of str or None, optional
+            Hatch pattern(s) to apply per dataset.
+
+            - If ``None`` (default), no hatching is applied.
+            - If a sequence of strings is provided (e.g., ``['//', 'xx', '..']``),
+              the patterns are cycled across datasets.
+            - Single string values (e.g., ``'//'``) are **not supported**.
+
+        Raises
+        ------
+        ValueError
+            If ``hatch`` is a single string or a non-iterable value.
+
```

Comment:
```suggestion
```

We don't list ValueErrors for invalid inputs under `Raises`. That would be an arbitrary and potentially long list with little benefit. It's by convention in Python that invalid inputs raise ValueErrors.